tools/tslint-rules/requireBreakingChangeVersionRule.ts

```diff
@@ -25,14 +28,28 @@ export class Rule extends Lint.Rules.AbstractRule {
           return;
         }
 
-        const hasBreakingChange = commentText.indexOf(BREAKING_CHANGE) > -1;
+        const breakingChangeIndex = commentText.indexOf(BREAKING_CHANGE);
+        const hasBreakingChange = breakingChangeIndex > -1;
+        const deprecatedIndex = commentText.indexOf(DEPRECATED);
+        const hasDeprecated = deprecatedIndex > -1;
 
-        if (!hasBreakingChange && commentText.indexOf('@deprecated') > -1) {
-          ctx.addFailure(pos, end, `@deprecated marker has to have a ${BREAKING_CHANGE}.`);
+        if (!hasBreakingChange && hasDeprecated) {
+          ctx.addFailure(pos, end, `${DEPRECATED} marker has to have a ${BREAKING_CHANGE}.`);
         } if (hasBreakingChange && !/\d+\.\d+\.\d+/.test(commentText)) {
```

Comment:
Should we have an `else if`? if not we should move the if to another line to make it more obvious whats happening here.

Reply:
ops, it should be an `else if`. The funny thing is that it's been there for ~2 years with nobody noticing.

Reply:
Its not really a problem since they are mutually exclusive since one is `hasBreakingChange` and one is `!hasBreakingChange`